tests/Invoke-DbaDbTransfer.Tests.ps1

```diff
@@ -0,0 +1,127 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [object[]]$params = (Get-Command $CommandName).Parameters.Keys
+        [object[]]$knownParameters = @(
+            'SqlInstance',
+            'SqlCredential',
+            'DestinationSqlInstance',
+            'DestinationSqlCredential',
+            'Database',
+            'DestinationDatabase',
+            'BatchSize',
+            'BulkCopyTimeOut',
+            'InputObject',
+            'EnableException',
+            'CopyAllObjects',
+            'CopyAll',
+            'SchemaOnly',
+            'DataOnly',
+            'ScriptingOption',
+            'ScriptOnly'
+        )
+        $knownParameters += [System.Management.Automation.PSCmdlet]::CommonParameters
+        It "Should only contain our specific parameters" {
+            $knownParameters | Where-Object { $_ } | Should -BeIn $params
+            $params | Should -BeIn ($knownParameters | Where-Object { $_ })
+        }
+    }
+}
+
+Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
+    BeforeAll {
+        $dbName = 'dbatools_transfer'
+        $source = Connect-DbaInstance -SqlInstance $script:instance1
+        $destination = Connect-DbaInstance -SqlInstance $script:instance2
```

Comment:
Please use instance 2 and 3 for this one....instance 1 & 2 have a good bit on them right now.

Reply:
Will do! Should I also put it into a different test group perhaps?

Reply:
The groupings are going to be reviewed again once we get done going through all the test adjustments/rewrites.